cluster-scope/base/groups/odh-pushgateway/group.yaml

```diff
@@ -0,0 +1,6 @@
+---
+apiVersion: user.openshift.io/v1
+kind: Group
+metadata:
+  name: odh-pushgateway
```

Comment:
Can we change this to something more general like odh or open-data-hub or Open-Data-Hub?

Reply:
hrm I'm worried this might get confusing because operate-first group also managing some odh namespaces under the opf-* umbrella, can you think of something else, like `odh-suffix`.

Reply:
How about ai-services?

Reply:
done